### PR TITLE
Add SNK file for strong name signing and update package versions

### PR DESCRIPTION
**Pull request by 96malhar**
*Description of changes:*
* Add SNK file to the repo for strong name signing.
* Update Aspire dependencies to 9.0.0
* Update the version number for `Aspire.Hosting.AWS` to 9.0.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <PackageTags>aspire integration hosting aws</PackageTags>
     <Description>Add support for provisioning AWS application resources and configuring the AWS SDK for .NET.</Description>
     <NoWarn>$(NoWarn);CS8002</NoWarn> <!-- AWS CDK packages are not signed -->
+    <SignAssembly>true</SignAssembly>
```
Comment (normj) [inline src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj:9]: Should we move the `SignAssembly` and `AssemblyOriginatorKeyFile` to `Directory.Build.props` so we don't forget to set them in the future?

File: tests/Aspire.Hosting.AWS.Tests/Aspire.Hosting.AWS.Tests.csproj [hunk 2]
```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
     <NoWarn>$(NoWarn);CS8002</NoWarn> <!-- AWS CDK packages are not signed -->
+    <AssemblyOriginatorKeyFile>..\..\public.snk</AssemblyOriginatorKeyFile>
```
Comment (normj) [inline tests/Aspire.Hosting.AWS.Tests/Aspire.Hosting.AWS.Tests.csproj:5]: Same comment as above for using `Directory.Build.props`.

**Review by normj — APPROVED** [review]

File: src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj (hunk 1 above)
Comment (96malhar) [reply src/Aspire.Hosting.AWS/Aspire.Hosting.AWS.csproj]: Done

File: tests/Aspire.Hosting.AWS.Tests/Aspire.Hosting.AWS.Tests.csproj (hunk 2 above)
Comment (96malhar) [reply tests/Aspire.Hosting.AWS.Tests/Aspire.Hosting.AWS.Tests.csproj]: Done

**Review by gcbeattyAWS — APPROVED** [review]

**Pull request merged**